modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableColumnHeader.java

```diff
@@ -455,7 +455,11 @@ void installTableColumnStyleClassListener() {
     NestedTableColumnHeader getNestedColumnHeader() { return nestedColumnHeader; }
     void setNestedColumnHeader(NestedTableColumnHeader nch) { nestedColumnHeader = nch; }
 
-    TableHeaderRow getTableHeaderRow() { return tableHeaderRow; }
+    /**
+     *
+     * @return The @link {@link TableHeaderRow} associated with this TableColumnHeader.
+     */
+    protected TableHeaderRow getTableHeaderRow() { return tableHeaderRow; }
```

Comment:
If anyone is tweaking the tableView header, it will tweak all the classes linked to it (TableHeaderRow created in TableViewSkinBase, NestedTableColumnHeader for rootHeader and NestedTableCOlumnHeader for actual column header).

If a custom TableHeaderRow has been provided in createTableHeaderRow() of TableViewSkinBase, it is only logical that its access should be protected in TableColumnHeader so that custom TableColumnHeader have access to the TableHeaderRow they're inside of.